de1plus/misc.tcl

```diff
@@ -686,16 +662,26 @@ proc make_de1_dir {srcdir destdirs} {
         plugins/DGUI/setup_DSx.tcl *
         plugins/DGUI/setup_Insight.tcl *
         plugins/DGUI/setup_MimojaCafe.tcl *
+    }
 
+    # Have skins deal with their own filelist if they want to
+    set skin_folders [lsort -dictionary [glob -nocomplain -tails -type d -directory "[homedir]/skins" * ]]
+
+    foreach s $skin_folders {
+        set fbasename [file rootname [file tail $s]]
+        if {[file exists "[homedir]/skins/$fbasename/filelist.txt"] == 1} {
```

Comment:
Personal preference as I start to get better with Tcl is to treat as a Boolean unless it is going to return something other than True False (or its equivalents). 

Here the meaning is clear. It is less clean with something like `$::settings(something) == 1` is that a Boolean, or what does "2" result in?

Reply:
@jeffsf I tend to use [string is true $::setttings(something)], which takes 1, yes, true, or otherwise false.